test/goldens/end2end/stream-2.0.1.json_report.md

```diff
@@ -53,10 +53,22 @@ No example found.
 See [package layout](https://dart.dev/tools/pub/package-layout#examples) guidelines on how to add an example.
 </details>
 
-## 10/20 Package is multi-platform
+## 0/20 Package is multi-platform
 
-### [~] 10/20 points: Supports 1 of 2 possible platforms (native, js)
+### [x] 0/20 points: Supports 0 of 2 possible platforms (native, js)
 
+<details>
+<summary>
+Package not compatible with runtime native-aot
+</summary>
+Because of the import of dart:mirrors via the import chain `package:stream/stream.dart` → `package:stream/plugin.dart` → `package:rikulo_commons/mirrors.dart` → `dart:mirrors`
```

Comment:
```
Because `dart:mirrors` is imported from:
 * `package:rikulo_commons/mirrors.dart`, which is imported from
 * `package:stream/plugin.dart`, which is imported from
 * `package:stream/stream.dart`.
```
Or maybe:
```
Because:
 * `package:stream/stream.dart`, which imports:
 * `package:stream/plugin.dart`, which imports:
 * `package:rikulo_commons/mirrors.dart`, which imports:
 * `dart:mirrors` which isn't supported on this platform.
```

Reply:
I'll do that in a separate PR. It touches the tagging part.